src/Abp/BackgroundJobs/IBackgroundJobConfiguration.cs

```diff
@@ -15,11 +16,17 @@ public interface IBackgroundJobConfiguration
         /// <summary>
         /// Period in milliseconds.
         /// </summary>
+        [Obsolete("Use UserTokenExpirationPeriod instead.")]
         int? CleanUserTokenPeriod { get; set; }
 
+        /// <summary>
+        /// Period for user token expiration.
```

Comment:
To disambiguate from user token expiration time:

```suggestion
        /// Period for user token expiration worker.
```